keyboards/keaboard/readme.md

```diff
@@ -0,0 +1,23 @@
+# keaboard
+
+![keaboard](https://raw.githubusercontent.com/keawade/keaboard/master/images/keaboard-rev1-pcb-render.png)
```

Comment:
This image needs to be rehosted on Imgur.

Reply:
I'll drop the image reference as I don't have an imgur account and I'm not interested in creating one as they require phone number confirmation.